selfservice/strategy/link/strategy_verification.go

```diff
@@ -323,8 +323,26 @@ func (s *Strategy) verificationUseToken(w http.ResponseWriter, r *http.Request,
 		return
 	}
 
-	http.Redirect(w, r, s.d.Config(r.Context()).SelfServiceFlowVerificationReturnTo(f.
-		AppendTo(s.d.Config(r.Context()).SelfServiceFlowVerificationUI())).String(), http.StatusFound)
+	for _, returnToKey := range []string{
+		"after_verification",
+		"", // empty string will revert to default
+	} {
```

Comment:
Why do we need to check two values here that have the same impact? Is it because `after_verification` (which IMO should be renamed to `after_verification_return_to`) is set at the registration endpoint, and it's being transferred over to this flow?

Reply:
Yes precisely. And happy to make that rename.

Reply:
Renamed

Reply:
refactored this